src/features/common/Layout/MyForestContextV2.tsx

```diff
@@ -52,15 +52,35 @@ interface MyForestContextV2Interface {
   contributionsMap: Map<string, MyContributionsMapItem> | undefined;
   registrationGeojson: RegistrationGeojson[];
   donationGeojson: DonationGeojson[];
+  treesPlanted: number;
+  areaRestored: number;
+  areaConserved: number;
+  isTargetModalLoading: boolean;
+  setIsTargetModalLoading: SetState<boolean>;
+  treeTarget: number;
+  restoreTarget: number;
+  setRestoreTarget: SetState<number>;
+  conservTarget: number;
+  setConservTarget: SetState<number>;
+  treeChecked: boolean;
+  setTreeChecked: SetState<boolean>;
+  restoreChecked: boolean;
+  setRestoreChecked: SetState<boolean>;
+  conservChecked: boolean;
+  setConservChecked: SetState<boolean>;
```

Comment:
Reconsider using context for everything, and duplicating data within the context. It makes your code more complicated and prone to bugs.

For example, `treesPlanted`, `areaRestored` etc. are simple calculations and can be easily done where needed instead of duplicating the data contained in `stats`. Why not use `stats` directly where needed?

`treeChecked`, `restoreChecked` --> why do these need to be in the context?

Reply:
I have cleaned up the context by removing all the states that can be managed at the component level.